terraform/helm/genesis/templates/genesis.yaml

```diff
@@ -0,0 +1,160 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "aptos-genesis.fullname" . }}-layout-e{{ .Values.chain.era }}
+  labels:
+    {{- include "aptos-genesis.labels" . | nindent 4 }}
+data:
+  layout.yaml: |-
+    ---
+    root_key: '0x5243ca72b0766d9e9cbf2debf6153443b01a1e0e6d086c7ea206eaf6f8043956'
```

Comment:
is that supposed to be hardcoded here>

Reply:
this key will be removed soon @gregnazario 
This is just the value that's been hardcoded in our other test scripts as well as (https://aptos.dev/tutorials/validator-node/run-validator-node-using-aws)